src/HandHistory.js

```diff
@@ -0,0 +1,148 @@
+class HandHistory {
+
+    constructor(player, cards) {
+        this.actions = [];
+        this.hero = player;
+        this.heroCards = cards;
+    }
+
+    // After you build one then for fuck's sake don't modify it. I'm trusting you.
+
+    // Actions
+
+    addStack(player, amount) {
+        this.actions.push({action: 'stack', player: player, amount: amount});
+    }
+
+    addBet(player, amount) {
```

Comment:
I am interested in the way the name aligns this with `addStack` rather than `fold` and `check`, but I may be overthinking things :-)

Reply:
Maybe "sit" is a better name. I sit down at the table with $400. All these methods could be named as verbs.

Reply:
Oh, that's interesting: those verbs are essentially the buttons I'm used to seeing in Puzzle Pirate's poker game